CONTRIBUTING.md

```diff
@@ -41,7 +41,20 @@ This is a rough outline of how to prepare a contribution:
 - Make sure any new code files have a license header.
 - Submit a pull request to the original repository.
 
-Thanks for your contributions!
+# Acceptance policy
+
+These things will make a PR more likely to be accepted:
+
+ * a well-described requirement
+ * tests for new code
+ * tests for old code!
+ * new code follows the conventions in old code
+
+In general, we will merge a PR once two maintainers have endorsed
+it. Trivial changes (e.g., corrections to spelling) may get waved
```

Comment:
We've been trying to stick to either one-sentence-per-line or one-paragraph-per-line with markdown files. Makes it easier to track changes.